File: src/msa/web-service/portalapi/src/main/java/vn/com/kns/portalapi/application/service/category/contact/dto/ContactDto.java
package vn.com.kns.portalapi.application.service.category.contact.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import vn.com.kns.portalapi.core.model.BaseEntityDto;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ContactDto extends BaseEntityDto {
    private String name;
    private String email;
    private String phone;
    private String title;//chức danh
    private String alias;//Ông/Bà
    private String contactType;//issue / order

//    private SupplierDto supplier;
    private Long supplierId;

//    private ShopDto shop;
    private Long shopId;
}

File: src/msa/web-service/portalweb/src/app/shared/services/contact/dto/contact-dto.ts
export class ContactDto {
  id: number | null;
  name: string;
  email: string;
  phone: string;
  title: string;
  alias: string;
  contactType: number;
  supplierId: number;
  shopId: number;

  constructor(id?: number | null, name?: string, shopId?: number | null, supplierId?: number | null) {
    this.id = id;
    this.name = name;
    this.shopId = shopId;
    this.supplierId = supplierId;
  }
}
